ReactWindows/ReactNative.Shared/UIManager/BorderedCanvas.cs

```diff
@@ -47,6 +52,307 @@ public Border Border
             }
         }
 
+        private class BorderProps
+        {
+            public Brush Brush;
+        }
+
+        private static readonly ThreadLocal<ConditionalWeakTable<BorderedCanvas, BorderProps>> s_borderProps =
```

Comment:
>BorderedCanvas [](start = 65, length = 14)

Why do you need a CWT here instead of just storing it as a field on the BorderedCanvas?